_studio/shared/umc/core/umc/include/umc_defs.h

```diff
@@ -34,22 +34,19 @@
 #if defined(LINUX32) && !defined(ANDROID)
 
     // video decoders
-    #define UMC_ENABLE_MJPEG_VIDEO_DECODER
-    #define UMC_ENABLE_VP8_VIDEO_DECODER
-    #define UMC_ENABLE_VC1_VIDEO_DECODER
-    #define UMC_ENABLE_H264_VIDEO_DECODER
-    #define UMC_ENABLE_H265_VIDEO_DECODER
-    #define UMC_ENABLE_MPEG2_VIDEO_DECODER
-    #define UMC_ENABLE_VP9_VIDEO_DECODER
+    #define MFX_ENABLE_VC1_VIDEO_DECODE
```

Comment:
no. Please, don't define same thing twice. We have this in mfx_config.h. Thus, include it here. Actually this file should contain single line (and copyright of course):

`#include <mfx_config.h>`